crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -31,15 +33,30 @@ impl Default for Visibility {
 }
 
 /// Algorithmically-computed indication of whether an entity is visible and should be extracted for rendering
-#[derive(Component, Clone, Reflect, Debug)]
-#[reflect(Component)]
+#[derive(Default, Debug)]
 pub struct ComputedVisibility {
-    pub is_visible: bool,
+    entities: FixedBitSet,
 }
 
-impl Default for ComputedVisibility {
-    fn default() -> Self {
-        Self { is_visible: true }
+impl ComputedVisibility {
+    #[inline]
+    pub fn mark_visible(&mut self, entity: Entity) {
+        self.entities.insert(entity.id() as usize);
+    }
+
+    #[inline]
+    pub fn is_visible(&self, entity: Entity) -> bool {
+        self.entities.contains(entity.id() as usize)
+    }
+
+    #[inline]
+    pub fn reserve(&mut self, entities: &Entities) {
```

Comment:
Why doesn't think just take a `usize` capacity?

Reply:
Ping? Because you closed and reopened the PR. :)

Reply:
Mostly just to force it to be inline with `Entities` of a given world. `FixedBitset` will panic if invariants aren't met, so it's best to ensure that the size comes from a source/type we know will provide a correct result.